6-guidelines.md

```diff
@@ -51,15 +51,15 @@ There is a clear distinction between "personal" and "space" projects. Keep this
 * So when using something, clean it afterwards and put it back in it's place.
 * If you are not trained to use tool X, don't use tool X but ask an expert to teach you first.
 * If you use one of the public workstations, please shut it off when you are done.
-* if you use the printer, please deposit some cash to pay for consumables
+* If you use the printer, please deposit some cash to pay for consumables.
 
 #### Damaging or losing space property
 
-If you damage or lose space property, you have to notice the community immediately in the Changelog or the Main channel on mattermost. Explain what happened and if/how you will fix it.
+If you damage or lose space property, you have to notice the community immediately in the Changelog or Main channel on mattermost. Explain what happened and if/how you will fix it.
 
 #### Taking space property out of the space
 
-Only members are allowed to take space property out of the space. If you take space property out of the space, you have to notify the community immediately, for example in the "changelog" or "main" channels on Mattermost. If a somebody disagrees with you taking it out of the space, make that person happy or put it back.
+Only members are allowed to take space property out of the space. If you take space property out of the space, you have to notify the community immediately, for example in the Changelog or Main channel on Mattermost. If a someone disagrees with you taking it out of the space, make that person happy or put it back.
```

Comment:
a someone -> someone ?

Reply:
somebody -> someone

Reply:
Please change "If a somebody disagrees" to "if someone disagrees". You left the "a" there, which shouldn't be there imo.